### PR TITLE
Login signup page edit

### PR DESCRIPTION
**Pull request by ifiokabasiudo**
Everythings been done with the login and signin, just style them and that's it

### CODE REVIEW

File: app/logo.png
Comment (Johnmiicheal) [inline app/logo.png:1]: You don't need this inside the app directory. All assets should be inside the public folder

File: src/components/Web/Footer.tsx
Comment (Johnmiicheal) [inline src/components/Web/Footer.tsx:1]: Add your github, rather than email...and as a link not plain text

**Review by Johnmiicheal — APPROVED** [review]
I'll test it locally...and style it

File: src/components/Web/Footer.tsx
Comment (ifiokabasiudo) [reply src/components/Web/Footer.tsx:1]: Oh yea, I forgot I could do that. I was thinking of a link I could use

File: pages/_app.tsx
Comment (Johnmiicheal) [inline pages/_app.tsx:1]: Bruhhhhhhhhhhhhhhh

Why did you delete this,,,,This is where the seo metadata is located na
